Tennis-scheduler/src/test/java/com/tennis/tennisscheduler/repositories/TennisCourtRepositoryTest.java

```diff
@@ -0,0 +1,42 @@
+package com.tennis.tennisscheduler.repositories;
+
+import com.tennis.tennisscheduler.models.Address;
+import com.tennis.tennisscheduler.models.TennisCourt;
+import com.tennis.tennisscheduler.models.enumes.SurfaceType;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@DataJpaTest
+class TennisCourtRepositoryTest {
+
+    @Autowired
+    TennisCourtRepository underTest;
```

Comment:
Please rename underTest to something that will say what it is when its called. Now I need to scroll up to check what underTest is